src/Symfony/Component/Form/Extension/Validator/ViolationMapper/ViolationMapper.php

```diff
@@ -188,7 +186,7 @@ private function matchChild(FormInterface $form, PropertyPathIteratorInterface $
             }
 
             /** @var FormInterface $child */
-            foreach ($children as $key => $child) {
+            foreach ($children as $i => $child) {
```

Comment:
Is there a specific reason for this change? If not, I like the original more. `$i` implies that it's being used for incrementation, but you're simply referring to the key here.